src/components/LabelMenus/index.js

```diff
@@ -1,70 +1,81 @@
+        },
+        {
+          title: "Use Cases and Alerts",
+          desc:
+            "Relevant alerts for data practitioners who intent to use this dataset for specific use cases (types of analyses).",
+        },
+        {
+          title: "Maintenance",
+          desc:
+            "Information about the ongoing management of the dataset, such as how the data will be maintained, updated, and the best contact for further inquiries.",
+        },
+      ],
     }
+    this.selectItem = this.selectItem.bind(this)
   }
 
-  selectItem = item =>
+  selectItem = item => {
     this.setState({
       selectedItem: item,
       showHighlight: false,
     })
+    console.log("item", item)
+    this.props.sendBaseInfo(item)
+  }
+
   render() {
     return (
-      <div>
-        <LabelTitle />
-        <div className={styles.flexComponents}>
-          <div
-            className={styles.labelMenu}
-            style={{ display: this.state.titleList }}
-          >
-            {this.state.titleList.map(title => {
-              return (
-                <div
-                  id="selectButton"
-                  key={title.id}
-                  onClick={() => this.selectItem(title)}
-                  className={
-                    this.state.selectedItem === title
-                      ? `${styles.highlightLabel} ${styles.labelMenus}`
-                      : styles.labelMenus
-                  }
-                >
-                  <div
-                    key={title.id}
-                    style={{
-                      display: this.state.showHighlight ? "block" : "none",
-                    }}
-                  ></div>
-                  <span className={styles.menuTitle}>{title}</span>
-                  <div className={styles.flexbox}>
-                    <p className={styles.labelMenuDescription}>Lorem</p>
-                    <span className={styles.weightedLine}></span>
-                  </div>
-                  <p className={styles.menuParagraph}>
-                    Qui ut occaecat exercitation amet mollit excepteur do
-                    aliquip et sunt sit aliqua consectetur. Sit minim voluptate
-                    aliqua id do velit deserunt. Voluptate irure proident esse
-                    sint duis mollit culpa eiusmod officia ullamco aliqua in
-                    magna dolore.
-                  </p>
-                </div>
-              )
-            })}
-          </div>
-          <SectionBase />
-        </div>
+      <div
+        className={styles.labelMenu}
+        style={{ display: this.state.titleList }}
+      >
+        {this.state.menus.map((menu, id) => {
+          return (
+            <div
+              id="selectButton"
+              onClick={() => this.selectItem(menu.title)}
+              className={
+                this.state.selectedItem === menu.title
+                  ? `${styles.highlightLabel} ${styles.labelMenus}`
+                  : styles.labelMenus
```

Comment:
For the styles conditional, let's use the `classNames()` function (see info [here](https://github.com/JedWatson/classnames#readme)). An example of how this might work here:
```jsx
className={classNames(styles.labelMenus, { [styles.highlightLabel]: (this.state.selectedItem === menu.title) } )}
```